### DISCUSSION TITLE
How are images associated with camera calibration?

### DISCUSSION THREAD

**Discussion by amacneil** [foxglove/community#609] [Q&A]
There aren't any links between RawImage/CompressedImage and CameraCalibration.

How does this work today? Should we add some place to reference a linked topic, or some concept similar to how `frame_id` works in 3D space?

**Comment by bhanug12** [comment]
A little digression. The ImageAnnotations still require a dummy (for the most part, except for height and width of the image) CameraCalibration. I think this can be better optimized by removing the dependency as it adds an extra field (CameraCalibration) that needs to be streamed/stored.

**Comment by amacneil** [comment]
@bhanug12 thanks for the feedback, I created a corresponding issue to track this in the studio repo as well: https://github.com/foxglove/community/issues/402
